tailwind.config.ts

```diff
@@ -67,6 +67,7 @@ const config = {
           DEFAULT: "hsl(var(--card))",
           foreground: "hsl(var(--card-foreground))",
         },
+        optimismRed: "#FF0420",
```

Comment:
this is weirdly in there as the "destructive" color rn if you wanna use that in your styling

Reply:
Oh weird ok I will rename that variable hahaha